.github/workflows/lint.yml

```
@@ -7,7 +7,7 @@ concurrency:
 on:
   pull_request:
   push:
-    branches: [24_1]
+    branches: [24_2]
```

```suggestion
    branches: [24_1, 24_2]
```
why removing 24_1 here and below?

---

The workflow can be different on branches 24_1 and 24_2. So we ensure that we run the right version of the workflow on push